.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
```

Comment:
`set -e` (too)?

Reply:
This script is going to execute in a k8s DaemonSet. If `set -e` and the script fails then the pod will end up on a "unpredictable state". The mechanism I employ to verify the installation succeeded (or not) is to write the return code of `kata-runtime kata-check` to `/tmp/kata_install_status` then put the pod to "sleep" (`tail -f /dev/null`). One control script will check the value of `/tmp/kata_install_status` on each pod, if all of them was done then the daemonSet is retired.

Let me know if I am missing something here, or if you think there is a better way to implement it.